config_example.sh

```diff
@@ -9,3 +9,15 @@ set -x
 # Uncomment to build a copy of ironic or inspector locally
 #export IRONIC_INSPECTOR_IMAGE=https://github.com/metalkube/metalkube-ironic-inspector
 #export IRONIC_IMAGE=https://github.com/metalkube/metalkube-ironic
+
+# Customize masters/workers specs
+# Memory in megabytes
+#export MASTER_MEMORY=6144
+# Num cpus
+#export MASTER_VCPU=2
+# Disk in gigabytes
+#export MASTER_DISK=50
+
+#export WORKER_MEMORY=6144
+#export WORKER_VCPU=8196
```

Comment:
typo - 8196 VCPU! :)

Reply:
Not a typo... don't you have such a powerful server? :P